kfs-core/src/main/java/org/kuali/kfs/fp/businessobject/ProcurementCardDefault.java

```diff
@@ -19,13 +19,19 @@
 package org.kuali.kfs.fp.businessobject;
```

Comment:
Property cardGroupId seems to have been renamed to reconcilerGroupId. 
Changes in property names from KFS 3.0 to KFS 6.0 will cause problems with the load of KFS 3.0 maintenance documents.

Property cardApprovalOfficial seems to be missing.

Property reconcilerGroupName has been added (not in KFS 3.0) Why?